### PR TITLE
Expand on the workflow

### PR DESCRIPTION
**Pull request by dragarok**

### CODE REVIEW

File: ContinuousML/NQ_DR_4_10_20_Ideal_27/1_Label_And_Feature_Workflow/Readme.md [hunk 1]
```diff
@@ -18,4 +18,5 @@
       - Host on dvc remote on GCP gsutil  
 - [x] .github/workflows/cml.yaml Globally...need to create a GitHub Actions workflow that calls dvc repro 
 # Links 
-[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1Zi6cdD2aPM9rf3bcUOPWdnp1GVaCfjsN?usp=sharing) Notebook exploring rudimentary feature selection techniques
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1Zi6cdD2aPM9rf3bcUOPWdnp1GVaCfjsN?usp=sharing) Notebook exploring rudimentary feature selection  techniques
```
Comment (Roger-Parkinson-EHP) [inline ContinuousML/NQ_DR_4_10_20_Ideal_27/1_Label_And_Feature_Workflow/Readme.md:21]: We can entirely replace this file with characterizations of feature importance.

File: ContinuousML/NQ_DR_4_10_20_Ideal_27/2_Training_Workflow/dvc.yaml [hunk 2]
```diff
@@ -12,4 +12,4 @@ stages:
     - ../../../src/Model_Training/Model_Training.py
     - Reduced_Features.csv
     outs:
-    - keras-model/
+    -   keras-model/
```
Comment (Roger-Parkinson-EHP) [inline ContinuousML/NQ_DR_4_10_20_Ideal_27/2_Training_Workflow/dvc.yaml]: This will be a machine learning model using ONNX format of *.onnx

File: ContinuousML/NQ_DR_4_10_20_Ideal_27/2_Training_Workflow/params.yaml [hunk 3]
```diff
@@ -7,4 +7,4 @@ train:
     activation: relu
     fc_layers: [512, 256, 128, 64]
 eval:
-    batch_size: 16
+    batch_size:   16
```
Comment (Roger-Parkinson-EHP) [inline ContinuousML/NQ_DR_4_10_20_Ideal_27/2_Training_Workflow/params.yaml:10]: We can enhance the hyperparameters here as teh model training process becomes more mature.

File: src/Feature_Importance/Feature_Importance.py [hunk 4]
```diff
@@ -99,7 +99,7 @@ def select_k_best_features_featurwiz(df, config):
 
     # Run feature selection
     selected_cols = select_k_best_features_sklearn(df, config)
-    # We need label as well for reduced feature used to train data later
+    #   We need label as well for reduced feature used to train data later
```
Comment (Roger-Parkinson-EHP) [inline src/Feature_Importance/Feature_Importance.py:106]: - [ ] The Reduced_feature.csv from the C# Feature Store will return a Label column that can be used for training. So you won't need to persist a labels.csv separately except during this testing.
- [ ] I do need this process to create a Selected_Features.csv that has one column with each row being a column name that passed your test. That file is read by the C# feature store in order to build the Reduced_Features.csv for training

Comment (Roger-Parkinson-EHP) [reply src/Feature_Importance/Feature_Importance.py:102]: I do need this process to create a Selected_Features.csv that has one column with each row being a column name that passed your test.

File: src/Model_Training/Model_Training.py [hunk 5]
```diff
@@ -8,7 +8,7 @@
 import yaml
 
 def run_model_training():
-    """Run model training using tensorflow
+    """  Run model training using tensorflow
```
Comment (Roger-Parkinson-EHP) [inline src/Model_Training/Model_Training.py:43]: - [ ] It would be great to use either keras tuner or optuna to optimize through the hyper parameters
- [x] As a starting point, I want to use F1 as a metric since this is an imbalanced dataset with a multi-class objective per:  https://twitter.com/haltakov/status/1440382566058496008
- [ ] It could be cool to populate a Readme.md with model characterizations. We need to figure out how this might fit in with the Plots and Metrics kept by CML though
- [x] The script needs to save the trained Model in ONNX format: https://www.codeproject.com/Articles/5278501/Making-Keras-Models-Portable-Using-ONNX
